### ISSUE TITLE
A problem about removeHead method in XcapStack.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BinaryBlue/imsdroid#147]
```
What steps will reproduce the problem?
1.I want to call removeHead method to remove a xcap message head,but the app is 
terminated...
2.
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
We are using IMSDroid/v1.1.327(doubango r519) on Android2.2 OS.

Please provide any additional information below.
Here is the source code I use to remove the head.
What should I do to remove a head successfully?
```

Original issue reported on code.google.com by `yuwenbin...@gmail.com` on 9 Jan 2011 at 5:08

Attachments:
- [source code.txt](https://storage.googleapis.com/google-code-attachments/imsdroid/issue-147/comment-0/source code.txt)

**Comment by GoogleCodeExporter** [comment]
```
The LogCat print "start to remove head!",but not print "finish to remove 
head!"...
```

Original comment by `yuwenbin...@gmail.com` on 9 Jan 2011 at 5:11

**Comment by GoogleCodeExporter** [comment]
Original comment by `boss...@yahoo.fr` on 10 Jan 2011 at 5:48
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.2.355.
```

Original comment by `boss...@yahoo.fr` on 11 Jan 2011 at 1:07
- Changed state: **Fixed**
